grab-x-link.el

```diff
@@ -1,45 +1,5 @@
-;;; grab-x-link.el --- Grab links from X11 apps and insert into Emacs  -*- lexical-binding: t; -*-
```

Comment:
The whole header section is required and should not be deleted.

Reply:
other changes look good to me.

Reply:
I kinda did that by accident. I'll put it back in tomorrow and push it up,